extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -536,7 +537,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.updateInViewport(this.slides_[
           dev().assertNumber(this.slideIndex_)], false);
     }
-    this.updateInViewport(this.slides_[newIndex], true);
+    const newSlideInView = this.slides_[newIndex];
+
+    if (newSlideInView === undefined) {
+      dev.error(
+        TAG,
+        'Accessing a non-existant slide at index: %s, noOfSlides: %s',
```

Comment:
I don't think errors support `%s` format, unfortunately. It's also best to construct an error with a non-var message and instead pass arguments in `Error.args` map.